src/kvstore/RocksEngineConfig.cpp

```diff
@@ -186,5 +194,27 @@ bool loadOptionsMap(std::unordered_map<std::string, std::string> &map, const std
     return true;
 }
 
+rocksdb::StatsLevel convertToStatsLevel(const std::string& str) {
+    if (str == "kExceptHistogramOrTimers") {
+        return rocksdb::StatsLevel::kExceptHistogramOrTimers;
+    } else if (str == "kExceptTimers") {
+        return rocksdb::StatsLevel::kExceptTimers;
+    } else if (str == "kExceptDetailedTimers") {
+        return rocksdb::StatsLevel::kExceptDetailedTimers;
+    } else if (str == "kExceptTimeForMutex") {
+        return rocksdb::StatsLevel::kExceptTimeForMutex;
+    } else {
+        return rocksdb::StatsLevel::kAll;
+    }
+}
+
+std::shared_ptr<rocksdb::Statistics> getDBStatistics() {
+    if (FLAGS_rocksdb_statistics_enable) {
+        static std::shared_ptr<rocksdb::Statistics> dbstats = rocksdb::CreateDBStatistics();
```

Comment:
Let rocksdb manage it's lifetime.

Reply:
We should pass the same Statistics object multiple DBs, in oder to aggregated values for all those DBs. so it's better to manage it ourselfs?

Reply:
If you share this cross db instances, the statistics may be mixed. The log of one db may contains other db's metrics data.

Reply:
Exactly true, IMHO dump to log is not the perfect way, we want report those metrics to the monitor system directly (such as Falcon), If do so, getDBStatistics is needed.
Can we disable rocksdb's dump feature and manage it ourself, I can do this in another PR.

Reply:
Even if so I suggest you manage the one DBStatistics object to one DB instance.

Reply:
You mean we'd better monitor each DB separately?

Reply:
Yes, I thinks so.

Reply:
I think there is no need, since they belong to the same space,  just different partitions only.

Reply:
No, there will be multiple DB instance in the storage process.

Reply:
Em, since we shared block_cache among all instance, so maybe we could collect stats of all instance.

Reply:
Ok, agree now. We could collect the all storage Statistics in one currently.